### PR TITLE
11500-create SprintBoot 22

### LINKED ISSUE #11500: Test Spring Boot 2.2.* application
Add a Spring Boot 2.2.* test application and add FAT tests for it.

### PR DESCRIPTION
**Pull request by Tonghui-Qu**
fixes #11500

### CODE REVIEW

**Comment by Tonghui-Qu** [comment]
#build

**Comment by anjumfatima90** [comment]
#build

**Comment by anjumfatima90** [comment]
#build

**Comment by anjumfatima90** [comment]
#build

File: dev/cnf/oss_dependencies.maven [hunk 1]
```diff
@@ -518,6 +518,16 @@ org.springframework:spring-webmvc:4.3.13.RELEASE
 org.springframework:spring-webmvc:5.0.8.RELEASE
 org.springframework:spring-webmvc:5.1.4.RELEASE
 org.springframework:spring-websocket:5.0.8.RELEASE
+org.springframework.security:spring-security-web:5.2.2.RELEASE
```
Comment (tevans78) [inline dev/cnf/oss_dependencies.maven]: please keep this file in alphabetical order so that we can avoid duplicates where possible

Comment (tevans78) [reply dev/cnf/oss_dependencies.maven]: in fact, there are already multiple versions of these dependencies ... please check if 5 different versions of spring-core (for example) are really needed?

**Comment by Tonghui-Qu** [comment]
#build

**Comment by anjumfatima90** [comment]
#build

File: dev/cnf/oss_dependencies.maven (hunk 1 above)
Comment (Tonghui-Qu) [reply dev/cnf/oss_dependencies.maven]: Thank you. I checked all the dependencies starting with "org.springframework". And found that there are 2 dependencies not used 
"org.springframework:spring-webmvc:4.3.13.RELEASE", 
"rg.springframework:spring-webflux:5.1.4.RELEASE"
And have make a commit to delete them. The other dependencies are used somewhere(we have several springboot app versions).

**Comment by anjumfatima90** [comment]
#build

**Comment by anjumfatima90** [comment]
#run-libby-bot

**Pull request merged**